plugins/assistant/api/assistantJobGeneral.js

```diff
@@ -35,69 +37,74 @@ else {
+                            feedDataIOS.anc_version = NOTIFICATION_VERSION;
+                            feedDataIOS.url = 'https://github.com/countly/countly-sdk-ios/releases.atom';
+                            feedDataIOS.targetHour = 12;
+                            dataForFeedMap.push(feedDataIOS);
+                        }
+
+                        // (3.3) New Android SDK release
+                        {
+                            let feedDataAndroid = {};
+                            feedDataAndroid.anc_i18n = "assistant.announcement-android-release";
+                            feedDataAndroid.anc_type = assistant.NOTIF_TYPE_ANNOUNCEMENTS;
+                            feedDataAndroid.anc_subtype = 3;
+                            feedDataAndroid.anc_version = NOTIFICATION_VERSION;
+                            feedDataAndroid.url = 'https://github.com/countly/countly-sdk-android/releases.atom';
+                            feedDataAndroid.targetHour = 13;
+                            dataForFeedMap.push(feedDataAndroid);
+                        }
+
+                        if (serverIsCE) {
+                            // (3.4) New community server release
+                            {
+                                let feedDataCommunity = {};
+                                feedDataCommunity.anc_i18n = "assistant.announcement-community-server-release";
+                                feedDataCommunity.anc_type = assistant.NOTIF_TYPE_ANNOUNCEMENTS;
+                                feedDataCommunity.anc_subtype = 4;
+                                feedDataCommunity.anc_version = NOTIFICATION_VERSION;
+                                feedDataCommunity.url = 'https://github.com/Countly/countly-server/releases.atom';
+                                feedDataCommunity.targetHour = 14;
+                                dataForFeedMap.push(feedDataCommunity);
+                            }
+                        }
+
+                        if (!serverIsCE) {
+                            // (3.5) New server code release for EE
+                            {
+                                let feedDataEE = {};
+                                feedDataEE.anc_i18n = "assistant.announcement-server-release-enterprise";
+                                feedDataEE.anc_type = assistant.NOTIF_TYPE_ANNOUNCEMENTS;
+                                feedDataEE.anc_subtype = 5;
+                                feedDataEE.anc_version = NOTIFICATION_VERSION;
+                                feedDataEE.url = 'https://github.com/Countly/countly-server/releases.atom';
+                                feedDataEE.targetHour = 15;
+                                dataForFeedMap.push(feedDataEE);
+                            }
+                        }
                     }
-                }
+                });
 
                 //go through all feeds
                 async.each(dataForFeedMap, function(feedItem, callbackOuter) {
```

Comment:
This creates a race condition, as `async.each(dataForFeedMap` will be run before `pluginManager.loadConfigs(db, function() {` will finish, so `dataForFeedMap` will never be populated. I guess you need to include all this into loadConfigs callback too.